internal/pkg/meta/constants.go

```diff
@@ -23,4 +23,8 @@ const (
 	UserReserved2
 	// UserReserved3 is reserved for user-defined metadata.
 	UserReserved3
+	// UUIDRewrite stores the UUID that this machine will use instead of the one from the hardware.
+	UUIDRewrite
+	// UniqueToken store the unique token for this machine. It's useful because UUID may repeat or be filled with zeros.
+	UniqueToken
```

Comment:
```suggestion
	UniqueMachineToken
```

might be a bit more clear what kind of token that is